README.md

```diff
@@ -277,16 +269,24 @@ outside of AWS. Run `scripts/create_config_aws_event_relay_user.py` to create an
 restricted access policy. This script also creates the user access key and stores it in an AWS Secrets Manager
 store.
 
-### User Authorization Configuration
-The following environment variables must be set to enable user authentication.
+## Running Tests
 
-* OIDC_AUDIENCE must be populated with the expected JWT (JSON web token) audience.
-* OPENID_PROVIDER is the generator of the JWT, and is used to determine how the JWT is validated.
-* OIDC_GROUP_CLAIM is the JWT claim that specifies the group the users belongs to.
-* OIDC_EMAIL_CLAIM is the JWT claim that specifies the requests email.
+1. Load sample data from the data-bundle-examples subrepository:
```

Comment:
This step can be taken out of the readme.. The `data-bundle-examples` sub-repo has been removed.